frontend/test/specs/homeControllerSpec.js

```diff
@@ -34,13 +34,14 @@
     beforeEach(module('app'));
 
     beforeEach(inject(function($controller, $httpBackend, $rootScope, $q, InstitutionService,
-            PostService, $mdDialog, $state, AuthService, $http) {
+            PostService, $mdDialog, $state, AuthService, $http, NotificationService) {
         httpBackend = $httpBackend;
         http = $http;
         scope = $rootScope.$new();
         mdDialog = $mdDialog;
         state = $state;
         postService = PostService;
+        notificationService = NotificationService;
```

Comment:
If the notificationsService mock is only being used within the beforeEach, you do not need to create a variable to access in the external scope because it already works for all tests.